lib/potassium/recipes/node.rb

```diff
@@ -0,0 +1,25 @@
+require 'net/http'
+require 'semantic'
+require 'pry'
+require 'json'
+
+class Recipes::Node < Rails::AppBuilder
+  def create
+    info "Using node version #{version_alias}"
+    create_file '.node-version', version_alias, force: true
+    js_package = JSON.load(Pathname.new("package.json"))
+    js_package["engines"]={"node"=> "#{version}"}
```

Comment:
Surrounding space missing for operator =.<br>Space inside { missing.<br>Surrounding space missing for operator =>.<br>Prefer to_s over string interpolation.<br>Space inside } missing.